content/en/docs/setup/turnkey/stackpoint.md

```diff
@@ -6,194 +6,180 @@ title: Running Kubernetes on Multiple Clouds with Stackpoint.io
 
 {{< toc >}}
 
-
 ## Introduction
 
-StackPointCloud is the universal control plane for Kubernetes Anywhere.  StackPointCloud allows you to deploy and manage a Kubernetes cluster to the cloud provider of your choice in 3 steps using a web-based interface.
+[StackPointCloud](https://stackpoint.io/) is the universal control plane for Kubernetes Anywhere. StackPointCloud allows you to deploy and manage a Kubernetes cluster to the cloud provider of your choice in 3 steps using a web-based interface.
+
 
 ## AWS
 
 To create a Kubernetes cluster on AWS, you will need an Access Key ID and a Secret Access Key from AWS.
 
-### Choose a Provider
-
-Log in to [stackpoint.io](https://stackpoint.io) with a GitHub, Google, or Twitter account.
-
-Click **+ADD A CLUSTER NOW**.
+1. Choose a Provider
 
-Click to select Amazon Web Services (AWS).
+    * Log in to [stackpoint.io](https://stackpoint.io) with a GitHub, Google, or Twitter account.
```

Comment:
@lucperkins You used star bullets for AWS provider installation steps, but letters for other providers.